content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -105,12 +79,10 @@ while (true)
 ```
    
 #### Methods used in ServerSocket
-
 -`public InputStream getInputStream()` - returns an InputStream object that receives data.
```

Comment:
Change:

> -`public InputStream getInputStream()` - returns an InputStream object that receives data.
> -`public OutputStream getOutputStream()`- returns an OutputStream for sending data.
> -`public Socket accept()` - Waits for client connection( the program won't continue until the client is connected). When you connect, you'll get a socket object that you may use to communicate with the client. 
> -`void close ()` - closes the server socket i.e stops waiting for requests from clients.

To:


> - `public InputStream getInputStream()` - returns an InputStream object that receives data.
> - `public OutputStream getOutputStream()`- returns an OutputStream for sending data.
> - `public Socket accept()` - Waits for client connection( the program won't continue until the client is connected). When you connect, you'll get a socket object that you may use to communicate with the client. 
> - `void close()` - closes the server socket i.e stops waiting for requests from clients.

You are missing a space between the *-* and the *`* for each item in the list. For example:

>-\`public OutputStream getOutputStream()`

should be:

>- \`public OutputStream getOutputStream()`

 This messes up the formatting of it a bit. Make sure to check things like these over in a markdown viewer. Also,